src/types/nitro.ts

```diff
@@ -31,11 +31,21 @@ export type NitroDynamicConfig = Pick<
   "runtimeConfig" | "routeRules"
 >;
 
-export interface NitroRuntimeConfig {
-  app: {
-    baseURL: string;
+export interface NitroRuntimeConfigApp {
```

Comment:
Maybe NitroAppRuntimeConfig? (I am fine with current one too if you prefer)

Reply:
Happy either way!